integ/pack.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+dir=$PWD
+filename=$(npm pack)
+
+echo "Copying $filename to $dir/../dist/js"
+cp "$filename" "$dir/../dist/js/"
```

Comment:
As discussed offline, there may have been concern that by adding packaging to the `integ` package, there may be side-effects on the publishing and API reference doc generation phases of our pipeline.

Adding this comment here to continue this discussion. The same feedback applies to `pkglint`.

Reply:
No longer packing these packages.